frontend/src/pages/CourseSelector/courseDescription/CourseDescription.jsx

```diff
@@ -26,6 +27,22 @@ const CourseAttribute = ({ title, content }) => {
   );
 };
 
+const ProgressBar = ({progress,height}) => {
+  let bgColor = "";
+  progress >= 75 ? bgColor = "red"
+  : progress >= 45 ? bgColor = "orange"
+  : progress >= 25 ? bgColor = "yellow"
+  : bgColor = "green";
+
+  return (
+  <div className="progressBarWrapper" height={height}>
```

Comment:
Nitpick: For the `height`, `width` and `backgroundColor` you could just use the `style` attributes and just create a style object to pass. i.e. `style={{height: height}}`. Just a personal preference tho 😄